docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+    reporter:
+        image: liberoadmin/reviewer-reporter:latest
```

Comment:
this image tag should come from https://github.com/elifesciences/elife-xpub-deployment/blob/develop/.env#L2-L4 (same value, `latest`, for now as it's the only one available)

Reply:
Changes made locally pending https://github.com/libero/reviewer-reporter/pull/2